Fix: step group documentation update and bug fix of firstExecuteTime mission (#3975)
kubevela/kubevela: docs/examples/workflow/step-group/README.md

# Step Group

## How to start

> Edit a yaml file as `example.yaml`, then execute it with `kubectl apply` command.

Reviewer: ```suggestion
Edit a yaml file as `example.yaml`, then execute it with `vela up` command.
```

## Parameter Introduction

> `step-group` have `subSteps` parameter which is an array contain any step type and its valid parameter exclude `step-group` step type itself

Reviewer: ```suggestion
`step-group` has a `subSteps` parameter which is an array containing any step type whose valid parameters do not include the `step-group` step type itself.
```

> `step-group` don't have `properties` parameter now

Reviewer: ```suggestion
Note that `step-group` doesn't support `properties` for now.
```

## Execute process

When executing the `step-group` step, the subSteps in the step group are executed in dag mode. The step group will only complete when all subSteps have been executed to completion.

> subStep have the same execute behaviour as it is a normal step

Reviewer: ```suggestion
SubStep has the same execution behavior as a normal step.
```

> ## Running result

Reviewer: I think we can delete this part and the `example-result.yaml` file.
